<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Backup</class>
 <widget class="QDialog" name="Backup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDC Backup</string>
  </property>
  <widget class="QProgressBar" name="backupProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup in progress. Please wait ...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Backup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
